zuul.d/arista-eos-jobs.yaml

```diff
@@ -69,213 +49,81 @@
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 2.7
-      ansible_test_split_in: 2
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable211-scenario01
-    parent: ansible-test-network-integration-eos-network_cli-python27-stable211
-    vars:
-      ansible_test_do_number: 1
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable211-scenario02
-    parent: ansible-test-network-integration-eos-network_cli-python27-stable211
-    vars:
-      ansible_test_do_number: 2
-
-- job:
-    name: ansible-test-network-integration-eos-httpapi-python27-stable29
-    parent: ansible-test-network-integration-eos-httpapi
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-eos-network_cli-python27-stable29
-    parent: ansible-test-network-integration-eos-network_cli
-    nodeset: eos-4.24.6-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-eos-python35
-    parent: ansible-test-network-integration-eos
-    nodeset: eos-4.24.6-python35
-    vars:
-      ansible_test_python: 3.5
+# =============================================================================
 
 - job:
-    name: ansible-test-network-integration-eos-python36
-    parent: ansible-test-network-integration-eos
-    nodeset: eos-4.24.6-python36
-    vars:
-      ansible_test_python: 3.6
+    name: network-ee-integration-tests-arista-eos
+    parent: ansible-network-eos-appliance
 
 - job:
-    name: ansible-test-network-integration-eos-httpapi-python36
-    parent: ansible-test-network-integration-eos-httpapi
-    nodeset: eos-4.24.6-python36
+    name: network-ee-integration-tests-arista-eos
+    parent: network-ee-integration-tests
+    nodeset: eos-4.24.6-python38
     vars:
-      ansible_test_python: 3.6
+      test_ansible_skip_tags: httpapi
 
 - job:
-    name: ansible-test-network-integration-eos-network_cli-python36
-    parent: ansible-test-network-integration-eos-network_cli
-    nodeset: eos-4.24.6-python36
+    name: network-ee-integration-tests-arista-eos-httpapi
+    parent: network-ee-integration-tests-arista-eos
     vars:
-      ansible_test_python: 3.6
+      test_ansible_skip_tags: network_cli
 
 - job:
-    name: ansible-test-network-integration-eos-network_cli-libssh-python36
-    parent: ansible-test-network-integration-eos-network_cli-python36
+    name: network-ee-integration-tests-arista-eos-libssh
+    parent: network-ee-integration-tests-arista-eos
     vars:
       ansible_test_network_cli_ssh_type: libssh
+      ansible_test_fips_mode: true
```

Comment:
we don't actually support this setting, so we can remove for now. I need to look up FIPS mode with podman / containers.

Reply:
actually, lets set this to test_enable_fips_mode: true. I'll work on network-ee patch